src/sasquatchbackpack/sasquatch.py

```diff
@@ -0,0 +1,127 @@
+"""Handles dispatch of backpack data to kafka."""
+
+import json
+from dataclasses import dataclass
+
+import requests
+
+from sasquatchbackpack import sources
+
+# Code yoinked from https://github.com/lsst-sqre/
+# sasquatch/blob/main/examples/RestProxyAPIExample.ipynb
+
+
+@dataclass
+class DispatcherConfig:
+    """Class containing relevant configuration information for the
+    BackpackDispatcher.
+    """
+
+    sasquatch_rest_proxy_url = (
+        "https://data-int.lsst.cloud/sasquatch-rest-proxy"
+    )
+
+
+class BackpackDispatcher:
+    """A class to send backpack data to kafka.
+
+    Parameters
+    ----------
+    source
+        DataSource containing schema and record data to be
+        posted to remote
+    config
+        DispatcherConfig to transmit other relevant information to
+        the Dispatcher
+    """
+
+    def __init__(
+        self, source: sources.DataSource, config: DispatcherConfig
+    ) -> None:
+        self.source = source
+        self.config = config
+        self.schema = source.load_schema()
+        self.namespace = self.get_namespace()
+
+    def get_namespace(self) -> str:
+        """Sorts the schema and returns the namespace value.
+
+        Returns
+        -------
+        namespace
+            provided namespace from the schema file
+        """
+        json_schema = json.loads(self.schema)
+
+        return json_schema["namespace"]
```

Comment:
We agreed that the namespace should be part of the `DispatcherConfig`, so this method is not necessary. Similar to  the other configuration parameters we need an environment variable for this too.

Reply:
If we want to be reading the namespae from the schema it sorta needs to be a dispatcher member. Are we scrapping that idea?